### PR TITLE
vvenc 1.9.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

- [x] `resource` blocks have been checked for updates.
<details>
  <summary>release notes</summary>
  <pre>* added rate capping during rate control (must be at least 1.5x of target rate)
* extensive improvements and cleanups to rate control, especially single pass (improved efficiency by 1-2%)
* speedup of first pass, especially for preset faster (around 15% overall runtime), using spatio-temporal downsampling
* unified MCTF activation for low- and high-rate encoding
* various speedups, fixes, and cleanups</pre>
</details>

### CODE REVIEW

**Comment by chenrui333** [comment]
`    Timeout::Error: execution expired`

File: Formula/vvenc.rb [hunk 1]
```diff
@@ -16,13 +16,9 @@ class Vvenc < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "e1cbbed66a84f67a944f46dfa1b56d32e6c20b45c4daca6286d1faadea6325b4"
   end
 
+  depends_on "ccache" => :build
```
Comment (carlocab) [inline Formula/vvenc.rb:19]: This is weird. Why is it needed? It seems like this would evade our compiler shims, maybe.

Comment (MikeMcQuaid) [reply Formula/vvenc.rb:19]: Good question. `ccache` won't actually do anything for us.

Comment (chenrui333) [reply Formula/vvenc.rb:19]: it complains in the build log, i can remove it and give it another try.

File: Formula/v/vvenc.rb [hunk 2]
```diff
@@ -33,14 +28,19 @@ def install
   end
 
   test do
-    resource("test_video").stage testpath
+    resource "homebrew-test_video" do
+      url "https://raw.githubusercontent.com/fraunhoferhhi/vvenc/master/test/data/RTn23_80x44p15_f15.yuv"
```
Comment (carlocab) [inline Formula/v/vvenc.rb:32]: This should probably not refer to `master`.

**Review by carlocab — APPROVED** [review]

**Pull request merged**